inc/endpoints/class-components-endpoint.php

```diff
@@ -92,8 +92,15 @@ public function __construct() {
 		add_filter( 'query_vars', [ $this, 'modify_query_vars' ] );
 		add_filter( 'post_row_actions', [ $this, 'add_api_link_to_posts' ], 10, 2 );
 		add_filter( 'page_row_actions', [ $this, 'add_api_link_to_posts' ], 10, 2 );
-		add_filter( 'tag_row_actions', [ $this, 'add_api_link_to_terms' ], 10, 2 );
 		add_filter( 'admin_bar_menu', [ $this, 'add_api_link_to_admin_bar' ], 999 );
+
+		// Set up taxonomy row actions.
+		// See: https://core.trac.wordpress.org/ticket/49808.
+		add_action( 'init', function () {
```

Comment:
I would recommend that the callback be moved to a named function within the class, similar to `[ $this, 'add_api_link_to_terms' ]`. That way, the function can be called directly in unit tests and is easier to debug.

Reply:
I normally I agree with making hook callbacks named functions. In this case all that function is doing is adding another callback to the correct hooks, so there's not really anything to test other than that the `add_api_link_to_terms` callbacks are being hooked.

I don't have a strong preference though, so I'll defer to your suggestion since I was on the fence about it.